### PR TITLE
Fix wrong keypair regeneration on upgrade

### LINKED ISSUE #5006: Zero token balance after switching to 7.4.0 branch.
##### Tribler version/branch+revision: #####

v7.4.0-exp3

##### Operating system and version: #####

Linux

##### Steps to reproduce the behavior: #####

Run Tribler 7.3.2. Then hiatus. Then switch to the v7.4.0 branch.

##### Expected behavior: #####

Token balance stays approximately the same as was left before.

##### Actual behavior: #####

"0 MBytes given to community, 0 MBytes taken from community".

##### Relevant log file output: #####

This gets logged repeatedly:

```
ERROR   1576450617.34                 community:695  (TriblerTunnelCommunity)  Exception occurred while handling packet!
Traceback (most recent call last):
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 692, in on_packet_from_circuit
    self.decode_map_private[msg_id](source_address, data, circuit_id)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 76, in wrapper
    return inner_wrapper(self, source_address, data)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/lazy_community.py", line 104, in wrapper
    return func(self, source_address, *unpacked)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 74, in inner_wrapper
    return func(inner_self, inner_source_address, *combo)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 774, in on_created
    self._ours_on_created_extended(circuit, payload)
  File "/home/tribler/tribler/Tribler/community/triblertunnel/community.py", line 447, in _ours_on_created_extended
    super(TriblerTunnelCommunity, self)._ours_on_created_extended(circuit, payload)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/community.py", line 551, in _ours_on_created_extended
    payload.auth, hop.public_key.key.pk)
  File "/home/tribler/tribler/Tribler/pyipv8/ipv8/messaging/anonymization/tunnelcrypto.py", line 51, in verify_and_generate_shared_secret
    libnacl.crypto_auth_verify(auth, dh_received, shared_secret)
  File "/home/tribler/.local/lib/python3.7/site-packages/libnacl/__init__.py", line 887, in crypto_auth_verify
    raise ValueError('Invalid secret key')
ValueError: Invalid secret key
```

### PR DESCRIPTION
**Pull request by ichorid**
fixes #5006

### CODE REVIEW

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

**Comment by ichorid** [comment]
retest this please

File: Tribler/Core/Session.py [hunk 1]
```diff
@@ -420,15 +419,16 @@ def start(self):
         def after_upgrade(_):
             self.upgrader = None
 
+
```
Comment (devos50) [inline Tribler/Core/Session.py]: Why was this newline added here? Methods require one newline between them right?

Comment (ichorid) [reply Tribler/Core/Session.py]: Just a mistype. I'm so used to `pre-commit` on `devel` that I don't consider formatting at all ))

Comment (ichorid) [reply Tribler/Core/Session.py]: @devos50 , I love how you care about formatting even on a branch that is not ever going to be merged in mainline :wink:

Comment (ichorid) [reply Tribler/Core/Session.py]: Line deleted.
Squash and merge at will.

**Comment by ichorid** [comment]
Line deleted.
Squash and merge at will.

**Pull request merged**